pkg/scheduler/extender_test.go

```diff
@@ -418,6 +417,7 @@ func TestIsInterested(t *testing.T) {
 }
 
 func TestConvertToMetaVictims(t *testing.T) {
+	t.Parallel()
```

Comment:
This seems irrelevant. Can we only focus on the replacement of `deepEqual` in this pr?

Reply:
Also, it seems that there are many changes of code style which results in a lot diff in git. 
Can we revert them as this pr is only focus on the replacement of deepEqual, may be we can do the style cleanup in another pr.

Reply:
You are right. I reverted it